app/services/katello/candlepin/consumer.rb

```diff
@@ -132,11 +132,18 @@ def compliance_reasons
 
       def self.orphaned_consumer_ids
         #returns consumer ids in candlepin with no matching katello entry
-        cp_consumers = ::Katello::Resources::Candlepin::Consumer.get({})
-        cp_consumers.reject! { |consumer| consumer['type']['label'] == 'uebercert' }
-        cp_consumer_ids = cp_consumers.map { |cons| cons["uuid"] }
-        katello_consumer_ids = ::Katello::Host::SubscriptionFacet.pluck(:uuid)
-        cp_consumer_ids - katello_consumer_ids
+        orphaned_ids = []
+        User.as_anonymous_admin do
+          cp_consumers = []
+          Organization.all.each do |org|
+            cp_consumers += ::Katello::Resources::Candlepin::Consumer.get('owner' => org.label)
+          end
+          cp_consumers.reject! { |consumer| consumer['type']['label'] == 'uebercert' }
+          cp_consumer_ids = cp_consumers.map { |cons| cons["uuid"] }
```

Comment:
`cp_consumers` is just an array of consumers right? The `cons` (while I get that its part of the word consumer) sounds plural and makes me think its multiple consumers. TLDR; can we just call it consumer inside the block? Easier to read IMO :)